### PR TITLE
Change examine function to include if item is a Clue Scroll item

### PR DESCRIPTION
**Pull request by critzd**
Simple feature included to the examine plugin to show if an item is used in any clue scrolls. Option to disable included in plugin, defaulted to true for now.

Showing inclusion on the Examine plugin config
java_2018-05-22_22-28-27

Showing textbox with results from examining item.
java_2018-05-22_22-28-50

Showing the lack of clue item text when plugin feature is disabled.
java_2018-05-22_22-29-54

### CODE REVIEW

**Comment by deathbeam** [comment]
This should be probably part of clue scroll plugin and reuse data from there.

File: runelite-client/src/main/java/net/runelite/client/plugins/examine/ExamineConfig.java [hunk 1]
```diff
@@ -79,4 +79,15 @@ default Color getTransparentExamineHRecolor()
 	{
 		return Color.decode("#9090FF");
 	}
+
+	@ConfigItem(
+			position = 6,
```
Comment (MESLewis) [inline runelite-client/src/main/java/net/runelite/client/plugins/examine/ExamineConfig.java]: Indent here is too great

**Comment by Adam-** [comment]
This is a neat idea. I think this probably belongs in the clue plugin though.

**Comment by Nightfirecat** [comment]
Per comments above, this functionality should be moved to the Clue plugin. As this PR has sat inactive for an exceedingly long time, I am closing it.

**Pull request closed without merging**